test/functional/test_runner.py

```diff
@@ -103,6 +103,7 @@
     'p2p_segwit.py',
     'feature_maxuploadtarget.py',
     'feature_assumeutxo.py',
+    'feature_reindex.py',
```

Comment:
What's the reason for moving this? If it's needed, could you add it to the commit message?

Reply:
Done.